src/convert_to_JSON/setup.py

```diff
@@ -0,0 +1,50 @@
+import json
+import sys
+from cohort import Cohort
+
+
+class ComplexEncoder(json.JSONEncoder):
```

Comment:
Can you explain why this is needed? We shouldn't have complex numbers anywhere in the output.